File: Pertemuan 5/operator.py
a = input("Masukkan nilai A = ")
b = input("Masukkan nilai B = ")

# Comparison Operators 
print("A == B = " + str(a == b))             
print("A != B = " + str(a != b))            
print("A > B = " + str(a > b))
print("A < B = " + str(a < b))
print("A >= B = " + str(a >= b))
print("A <= B = " + str(a <= b))

# Logical Operators
print("A < 6 and B > 2 = " + str(int(a) < 6 and int(b) > 2))
print("A < 4 or B > 3 = " + str(int(a) < 4 or int(b) > 3))
print(" not(A < 7) and not(B > 5) = " + str(not(int(a) < 7) and not(int(b) > 5)))

File: Pertemuan 5/Operator.java
import java.util.Scanner;

public class Operator {
    public static void main (String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Masukkan nilai A: ");
        double a = scanner.nextDouble();
        System.out.print("Masukkan nilai B: ");
        double b = scanner.nextDouble();

        // Comparison Operators
        System.out.println("A == B " + (a == b));       
        System.out.println("A != B " + (a != b));
        System.out.println("A > B " + (a > b));
        System.out.println("A < B " + (a < b));
        System.out.println("A >= B " + (a >= b));
        System.out.println("A <= B " + (a <= b));

        // Logical Operators
        System.out.println("Hasil A < 10 && B > 6 = " + (a < 10 && b > 6));     // Logical and (&&)
        System.out.println("Hasil A < 12 || B > 9 = " + (a < 12 || b > 9));     // Logical or (||)
        System.out.println("Hasil !(A < 8 && B > 5) = " + !(a < 8 && b > 5));   // Logical not (!)
    }
}